src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/AttributePresenceFilterNode.cs

```diff
@@ -101,6 +102,100 @@ private List<CustomAttributeEntry> GetCustomAttributeEntries()
             return customAttributeEntries;
         }
 
+        private class FakeType
+        {
+            public static FakeType Instance = new FakeType();
+        }
+
+        private class FakeGenericContext
+        {
+        }
+
+        private class FakeSignatureTypeProvider : ISignatureTypeProvider<FakeType, FakeGenericContext>
```

Comment:
I don't like the Fake name. Come up with something that describes what's happening here. Put in a comment, and then I approve.